app/upgrades/v24/constants.go

```diff
@@ -19,3 +19,750 @@ var Upgrade = upgrades.Upgrade{
 		Deleted: []string{},
 	},
 }
+
+var FinalIncentiveAccumulatorPoolIDsToMigrate = map[uint64]struct{}{
```

Comment:
What if new someone creates new CL pools before we release v24?

Would it make sense to make the system more dynamic? That is, migrate every CL pool other than the hard-coded white-listed list which is guaranteed to not change?

Reply:
It wouldn't matter because after v23, all future pools created are on the new upscaled accumulator logic right?

Reply:
I think it's fine, also based on the [Threshold](https://github.com/osmosis-labs/osmosis/pull/7805/files#diff-60e2a1beda87745518eb040a91479246d2073a318fd5b5d2f20aabf9755e7c07R125-R127) we will only ever migrate up to pool 1492

```
pool_id=1492
```

so all newly created pool are using the new scaling factor

Reply:
But I've added all the latest CL pools to the migration, although they will be a NO-OP